src/api/models/profiles/index.ts

```diff
@@ -0,0 +1,7 @@
+// exports all functions out of the folder to profiles.route.ts and elsewhere
```

Comment:
It would proabably be more correct to say something like
```
// exports all profile functions from this folder's path
```

We export here so that we can import them to `profiles.route.ts`, but nothing in this file actually does that import